eng/Versions.props

```diff
@@ -34,7 +34,7 @@
     <VisualStudioEditorPackagesVersion>16.9.220</VisualStudioEditorPackagesVersion>
     <ILAsmPackageVersion>5.0.0-alpha1.19409.1</ILAsmPackageVersion>
     <ILDAsmPackageVersion>5.0.0-preview.1.20112.8</ILDAsmPackageVersion>
-    <MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>16.9.150</MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>
+    <MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>16.10.139</MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>
```

Comment:
Does this need to go in vs-deps?

Reply:
Not sure what that is 😄

Reply:
@davidwengier @NTaylorMullen this does not - we ship a copy of the lsp protocol packages, we don't rely on one from VS.

Typically we use the -vs-deps branch for changes that upgrade dependencies to versions not present in a public VS release so that roslyn main can be used with the latest non-preview version.